compass/landice/tests/ismip6_forcing/ismip6_forcing.cfg

```diff
@@ -0,0 +1,58 @@
+# config options for ismip6 antarctic ice sheet data set
+[ismip6_ais]
+
+# Base path to the input ismip6 ocean and smb forcing files. User has to supply.
+base_path_ismip6 = NotAvailable
+
+# Base path to the the MALI mesh. User has to supply.
+base_path_mali = NotAvailable
+
+# Forcing end year of the ISMIP6 data. User has to supply.
+# Available end years are 2100 and 2300.
+period_endyear = NotAvailable
+
+# Base path to which output forcing files are saved.
+output_base_path = NotAvailable
+
+# Name of climate model name used to generate ISMIP6 forcing data. User has to supply.
+# Available model names are the following: CCSM4, CESM2-WACCM, CSIRO-Mk3-6-0, HadGEM2-ES, NorESM1-M, UKESM1-0-LL
+model = NotAvailable
+
+# Scenarios used by climate model. User has to supply.
+# Available scenarios are the following: RCP26, RCP85, SSP126, SSP585
+scenario = NotAvailable
+
+# name of the mali mesh. User has to supply. Note: It is used to name mapping files
+# (e,g. 'map_ismip6_8km_to_{mali_mesh_name}_{method_remap}.nc').
+mali_mesh_name = NotAvailable
+
+# MALI mesh file to be used to build mapping file (e.g.Antarctic_8to80km_20220407.nc). User has to supply.
+mali_mesh_file = NotAvailable
+
+# config options for ismip6 antarctic ice sheet SMB forcing data test cases
+[ismip6_ais_atmosphere]
+
+# Remapping method used in building a mapping file. Options include: bilinear, neareststod, conserve
+method_remap = bilinear
+
+# Set True to process RACMO modern climatology 
+process_smb_racmo = True
+
+# Full path to the RACMO regional climate model SMB data for modern climatology
+base_path_racmo = NotAvailable
+
+# config options for ismip6 ocean thermal forcing data test cases
+[ismip6_ais_ocean_thermal]
+
+# Remapping method used in building a mapping file. Options include: bilinear, neareststod, conserve
+method_remap = bilinear
+
+# Set to True if want to process observational thermal forcing data. Set to False if want to process model thermal forcing data.
+process_obs_data = False
+
+# config options for ismip6 ocean basal test case
+[ismip6_ais_ocean_basal]
+
+# Remapping method used in building a mapping file. Note: ocean basal testcase will always want to
+# use neareststod method.
+method_remap = neareststod
```

Comment:
We agreed to get rid of this config option (and therefore the section) because a user doesn't have a choice here.